src/data-mgt/node/controllers/transform.js

```diff
@@ -103,7 +103,7 @@ const data = {
                   cacheID,
                   JSON.stringify({ isCache: true, ...responseData })
                 );
-                redis.expire(cacheID, 86400);
+                redis.expire(cacheID, 30);
```

Comment:
could this be a typo? is it 30 or 300??

Reply:
This was intentional, different endpoints have different cache expiry times...

Reply:
Okay, got it. thanks!